astropy/units/quantity.py

```diff
@@ -266,7 +266,12 @@ class Quantity(np.ndarray):
     TypeError
         If the unit provided is not either a :class:`~astropy.units.Unit`
         object or a parseable string unit.
-
+        
+    Warnings
```

Comment:
I think `.. warning::` at the top might give you a nice red box. But if this is clear enough, I guess that is okay too. Will have to see how RTD renders this.

Are you able to address the PEP 8 and squash? Thanks!

Reply:
Sorry, that's what I get by using the web interface 😆 . Will fix in a bit.